api/controllers/roomController.js

```diff
@@ -0,0 +1,12 @@
+import {queryByAsync} from "../services/index.js";
```

Comment:
queryByAsync 라는 함수명은 뭘까요?
controller 함수명은 잘 바꾸었습니다. 그에 맞게 service, dao도 수정해 주시면 좋을 것 같습니다.

그리고 리나님도 room쪽 하시는 것 같던데, 파일명 통일 시킬 수 있게 논의해 주세요.
```suggestion
import { queryByAsync } from "../services/index.js";
```